DNN Platform/Library/Services/Personalization/PersonalizationController.cs

```diff
@@ -174,9 +174,9 @@ private static string DecryptData(string profileData)
         private static string GetDecryptionkey()
         {
             var machineKey = Config.GetDecryptionkey();
-            var hostGuid = Host.GUID.Replace("-", string.Empty);
-            var key = (machineKey ?? "") + hostGuid;
-            return PortalSecurity.Instance.Encrypt(key, key);
+            var hostGuid = Host.GUID;
```

Comment:
If changing the way this is composed, do away with this declaration as a string it is immutable so no reason to actually allocate this item.  Just use Host.GUID directly below in the String Interpolation call